static/templates/search_result.html

```diff
@@ -137,32 +137,32 @@
     </div>
 
     <div class="footer">
-      {{#hasPages}}
+      {{#pagination.hasPages}}
         <ul>
-          {{#resultLastPageStart}}
+          {{#pagination.firstPage}}
             <li>
-              <a href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start=0&pageLength={{pageLength}}">
+              <a href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start={{start}}&pageLength={{pagination.itemsPerPage}}">
                 ◀
               </a>
             </li>
-          {{/resultLastPageStart}}
-          {{#pages}}
+          {{/pagination.firstPage}}
+          {{#pagination.pages}}
             <li>
-              <a {{#selected}}class="selected"{{/selected}}
-                 href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start={{start}}&pageLength={{pageLength}}">
+              <a {{#current}}class="selected"{{/current}}
+                 href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start={{start}}&pageLength={{pagination.itemsPerPage}}">
                 {{label}}
               </a>
             </li>
-          {{/pages}}
-          {{#resultLastPageStart}}
+          {{/pagination.pages}}
+          {{#pagination.lastPage}}
             <li>
-              <a href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start={{resultLastPageStart}}&pageLength={{pageLength}}">
+              <a href="{{searchProtocolPrefix}}pattern={{searchPatternEncoded}}{{#contentId}}&content={{.}}{{/contentId}}&start={{start}}&pageLength={{pagination.itemsPerPage}}">
                 ▶
               </a>
             </li>
-          {{/resultLastPageStart}}
+          {{/pagination.lastPage}}
         </ul>
-      {{/hasPages}}
+      {{/pagination.hasPages}}
```

Comment:
Doesn't it make sense to simplify this template by removing the special handling of the goto first and last page triangle markers (`◀`, `▶`) but instead including them (conditionally) as labels (with corresponding `start` values) in the regular list of `pagination.pages`?

Reply:
Done